backend/src/app.module.ts

```
@@ -56,6 +56,8 @@ import { RequestLogModule } from './request_log/request_log.module';
 import { RunReportModule } from './task/run_report/run-report.module';
 import { TaskModule } from './task/task.module';
 import { ClassificacaoModule } from './transferencias-voluntarias/classificacao/classificacao.module';
+import { PSMFDashboardModule } from './mf/ps-dash/ps-dash.module';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'PSMFDashboardModule'. Consider to delete it or rename it.

'PSMFDashboardModule' is defined here:


(lint/suspicious/noRedeclare)